apps/web/src/components/terminal/TerminalErrorBoundary.tsx

```
@@ -10,73 +11,63 @@ interface Props {
 
-          <button
-            onClick={this.toggleDetails}
-            className="flex items-center gap-1 text-xs text-muted-foreground hover:text-foreground transition-colors"
-          >
-            {this.state.showDetails ? <ChevronUp size={12} /> : <ChevronDown size={12} />}
-            Technical Details
-          </button>
+      <button
+        onClick={handleRestart}
+        className="flex items-center gap-2 px-4 py-2 bg-primary text-primary-foreground rounded-md hover:bg-primary/90 transition-colors text-sm"
+      >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)